bin/psysh

```diff
@@ -82,7 +82,6 @@ call_user_func(function() {
 // We didn't find an autoloader for a local version, so use the autoloader that
 // came with this script.
 if (!class_exists('\Psy\Shell')) {
-/* <<< */
```

Comment:
These comments are necessary, they're how the compiler knows where to put the phar autoloader code.